hwy/tests/reverse_test.cc

```diff
@@ -217,9 +217,7 @@ class TestReverseBits {
 };
 
 HWY_NOINLINE void TestAllReverse() {
-  // 8-bit is not supported because Risc-V uses rgather of Lanes - Iota,
-  // which requires 16 bits.
-  ForUIF163264(ForPartialVectors<TestReverse>());
+  ForAllTypes(ForPartialVectors<TestReverse>());
```

Comment:
I think this is still unresolved. The new test looks fine, but there was and is a comment in the rvv file:
```
// <32bit are not part of Highway API, but used in Broadcast. This limits VLMAX
// to 2048! We could instead use vrgatherei16.
#define HWY_RVV_TABLE(BASE, CHAR, SEW, SEWD, SEWH, LMUL, LMULD, LMULH, SHIFT, \
```
<32 bit types are now indeed a part of the API, but the issue remains that we would want to use HWY_RVV_TABLE16 for 8-bit types to safely handle >2048-bit vectors (which can happen on BSC hardware).
Would you like to give that a go, or shall I do it after we merge?

Reply:
Ah, thanks for pointing that out. So the only thing that requires update is the comment on HWY_RVV_TABLE; I can do that after we merge.